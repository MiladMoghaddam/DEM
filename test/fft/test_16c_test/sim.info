{'cmdline': ['../../run-sniper',
             '-n',
             '16',
             '-c',
             'gainestown',
             '-c',
             'noc',
             '--roi',
             '-d',
             './test_16c_test',
             '-speriodic-stats:10000:0',
             '-senergystats:10000',
             '-g',
             '--perf_model/core/frequency=2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2',
             '--',
             './fft',
             '-p',
             '16'],
 'git_revision': '593709afd5ac1a5756c123e74e437122d9b328a1',
 'host': 'sealion',
 'pin_version': 71313,
 'rusage': (46.324,
            11.04,
            120728,
            0,
            0,
            0,
            13934510,
            0,
            0,
            0,
            91376,
            0,
            0,
            0,
            57228,
            211448),
 'snipercmd': ['bash',
               '-c',
               '/home/milad/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /home/milad/sniper/lib/pin_sim -c /home/milad/sniper/config/base.cfg --general/total_cores=16 --general/output_dir=/home/milad/sniper/test/fft/test_16c_test --config=/home/milad/sniper/config/nehalem.cfg --config=/home/milad/sniper/config/gainestown.cfg --config=/home/milad/sniper/config/noc.cfg --perf_model/core/frequency=2,2,2,2,2,2,2,2,2,2,2,2,2,2,2,2 -g --general/magic=true -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/home/milad/sniper/test/fft/test_16c_test/sim.scripts.py -g --hooks/script0args= -- ./fft -p 16'],
 't_elapsed': 74.53991889953613,
 't_start': 1484078579.851483,
 'tracecmds': [],
 'user': 'milad',
 'vmem': 2339893248L}
