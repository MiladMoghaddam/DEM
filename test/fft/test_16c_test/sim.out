                                     | Core 0     | Core 1     | Core 2     | Core 3     | Core 4     | Core 5     | Core 6     | Core 7     | Core 8     | Core 9     | Core 10    | Core 11    | Core 12    | Core 13    | Core 14    | Core 15   
  Instructions                       |      80982 |      69328 |      69419 |      69282 |      69448 |      69409 |      69372 |      69388 |      69270 |      69472 |      69454 |      69570 |      69341 |      69446 |      69327 |      69149
  Cycles                             |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477 |     587477
  IPC                                |       0.14 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12 |       0.12
  Time (ns)                          |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738 |     293738
  Idle time (ns)                     |     246527 |     252739 |     253378 |     251143 |     252886 |     250594 |     249698 |     247570 |     249960 |     249594 |     251353 |     248814 |     251502 |     248516 |     248456 |     251063
  Idle time (%)                      |      83.9% |      86.0% |      86.3% |      85.5% |      86.1% |      85.3% |      85.0% |      84.3% |      85.1% |      85.0% |      85.6% |      84.7% |      85.6% |      84.6% |      84.6% |      85.5%
Branch predictor stats               |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num correct                        |       4484 |       2931 |       2917 |       2925 |       2923 |       2922 |       2920 |       2925 |       2891 |       2931 |       2931 |       2920 |       2900 |       2899 |       2902 |       2864
  num incorrect                      |        359 |        327 |        338 |        322 |        333 |        332 |        330 |        328 |        335 |        338 |        327 |        347 |        334 |        348 |        336 |        333
  misprediction rate                 |      7.41% |     10.04% |     10.38% |      9.92% |     10.23% |     10.20% |     10.15% |     10.08% |     10.38% |     10.34% |     10.04% |     10.62% |     10.33% |     10.72% |     10.38% |     10.42%
  mpki                               |       4.43 |       4.72 |       4.87 |       4.65 |       4.79 |       4.78 |       4.76 |       4.73 |       4.84 |       4.87 |       4.71 |       4.99 |       4.82 |       5.01 |       4.85 |       4.82
TLB Summary                          |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  I-TLB                              |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |       7700 |       6176 |       6211 |       6174 |       6227 |       6214 |       6207 |       6215 |       6203 |       6235 |       6244 |       6278 |       6235 |       6261 |       6231 |       6211
    num misses                       |          5 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29 |         29
    miss rate                        |      0.06% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.47% |      0.46% |      0.46% |      0.47% |      0.46% |      0.47% |      0.47%
    mpki                             |       0.06 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42 |       0.42
  D-TLB                              |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |      51714 |      46751 |      46793 |      46735 |      46805 |      46792 |      46780 |      46781 |      46735 |      46815 |      46804 |      46853 |      46758 |      46806 |      46756 |      46690
    num misses                       |         75 |         48 |         48 |         48 |         48 |         48 |         48 |         48 |         48 |         50 |         48 |         48 |         48 |         48 |         48 |         48
    miss rate                        |      0.15% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.11% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10% |      0.10%
    mpki                             |       0.93 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.72 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69 |       0.69
  L2 TLB                             |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num accesses                     |         80 |         77 |         77 |         77 |         77 |         77 |         77 |         77 |         77 |         79 |         77 |         77 |         77 |         77 |         77 |         77
    num misses                       |         51 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76 |         76
    miss rate                        |     63.75% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     96.20% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70% |     98.70%
    mpki                             |       0.63 |       1.10 |       1.09 |       1.10 |       1.09 |       1.09 |       1.10 |       1.10 |       1.10 |       1.09 |       1.09 |       1.09 |       1.10 |       1.09 |       1.10 |       1.10
Cache Summary                        |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  Cache L1-I                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |       7700 |       6176 |       6211 |       6174 |       6227 |       6214 |       6207 |       6215 |       6203 |       6235 |       6244 |       6278 |       6235 |       6261 |       6231 |       6211
    num cache misses                 |        218 |        269 |        270 |        267 |        270 |        266 |        266 |        268 |        273 |        268 |        269 |        270 |        273 |        273 |        274 |        269
    miss rate                        |      2.83% |      4.36% |      4.35% |      4.32% |      4.34% |      4.28% |      4.29% |      4.31% |      4.40% |      4.30% |      4.31% |      4.30% |      4.38% |      4.36% |      4.40% |      4.33%
    mpki                             |       2.69 |       3.88 |       3.89 |       3.85 |       3.89 |       3.83 |       3.83 |       3.86 |       3.94 |       3.86 |       3.87 |       3.88 |       3.94 |       3.93 |       3.95 |       3.89
  Cache L1-D                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |      51997 |      46855 |      46898 |      46836 |      46910 |      46898 |      46884 |      46886 |      46835 |      46926 |      46910 |      46964 |      46861 |      46913 |      46857 |      46783
    num cache misses                 |        585 |        314 |        332 |        314 |        327 |        330 |        332 |        336 |        340 |        340 |        331 |        353 |        345 |        339 |        340 |        347
    miss rate                        |      1.13% |      0.67% |      0.71% |      0.67% |      0.70% |      0.70% |      0.71% |      0.72% |      0.73% |      0.72% |      0.71% |      0.75% |      0.74% |      0.72% |      0.73% |      0.74%
    mpki                             |       7.22 |       4.53 |       4.78 |       4.53 |       4.71 |       4.75 |       4.79 |       4.84 |       4.91 |       4.89 |       4.77 |       5.07 |       4.98 |       4.88 |       4.90 |       5.02
  Cache L2                           |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |        882 |        642 |        673 |        641 |        670 |        663 |        668 |        674 |        688 |        671 |        673 |        692 |        699 |        680 |        685 |        682
    num cache misses                 |        657 |        569 |        588 |        567 |        582 |        580 |        582 |        588 |        597 |        592 |        584 |        607 |        602 |        596 |        595 |        597
    miss rate                        |     74.49% |     88.63% |     87.37% |     88.46% |     86.87% |     87.48% |     87.13% |     87.24% |     86.77% |     88.23% |     86.78% |     87.72% |     86.12% |     87.65% |     86.86% |     87.54%
    mpki                             |       8.11 |       8.21 |       8.47 |       8.18 |       8.38 |       8.36 |       8.39 |       8.47 |       8.62 |       8.52 |       8.41 |       8.73 |       8.68 |       8.58 |       8.58 |       8.63
  Cache L3                           |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
    num cache accesses               |        750 |        632 |        659 |        633 |        655 |        653 |        652 |        664 |        672 |        661 |        657 |        682 |        683 |        670 |        666 |        669
    num cache misses                 |        488 |        231 |        170 |        225 |        244 |        296 |        251 |        376 |        402 |        214 |        241 |        339 |        285 |        320 |        329 |        270
    miss rate                        |     65.07% |     36.55% |     25.80% |     35.55% |     37.25% |     45.33% |     38.50% |     56.63% |     59.82% |     32.38% |     36.68% |     49.71% |     41.73% |     47.76% |     49.40% |     40.36%
    mpki                             |       6.03 |       3.33 |       2.45 |       3.25 |       3.51 |       4.26 |       3.62 |       5.42 |       5.80 |       3.08 |       3.47 |       4.87 |       4.11 |       4.61 |       4.75 |       3.90
DRAM summary                         |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num dram accesses                  |        661 |          0 |          0 |          0 |        634 |          0 |          0 |          0 |        598 |          0 |          0 |          0 |        619 |          0 |          0 |          0
  average dram access latency (ns)   |      70.26 |        inf |        inf |        inf |      58.22 |        inf |        inf |        inf |      57.13 |        inf |        inf |        inf |      57.46 |        inf |        inf |        inf
  average dram queueing delay        |      16.83 |       0.00 |       0.00 |       0.00 |       4.79 |       0.00 |       0.00 |       0.00 |       3.70 |       0.00 |       0.00 |       0.00 |       4.03 |       0.00 |       0.00 |       0.00
  average dram bandwidth utilization |      1.00% |      0.00% |      0.00% |      0.00% |      1.00% |      0.00% |      0.00% |      0.00% |      1.00% |      0.00% |      0.00% |      0.00% |      1.00% |      0.00% |      0.00% |      0.00%
Coherency Traffic                    |            |            |            |            |            |            |            |            |            |            |            |            |            |            |            |           
  num loads from dram                |         25 |         29 |         29 |         28 |         26 |         43 |         36 |         44 |         56 |         31 |         33 |         44 |         38 |         45 |         39 |         28
  num loads from dram cache          |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0 |          0
  num loads from remote cache        |         77 |         44 |         32 |         67 |         74 |        126 |         56 |        128 |        134 |         67 |         64 |        134 |         77 |        120 |        116 |         99
